posthog/api/team.py

```diff
@@ -95,6 +96,8 @@ def validate(self, attrs: Any) -> Any:
         if "primary_dashboard" in attrs and attrs["primary_dashboard"].team != self.instance:
             raise exceptions.PermissionDenied("Dashboard does not belong to this team.")
 
+        # TODO: Add validation here for person_display_name_properties to be list of strings?
```

Comment:
@mariusandra I didn't see a massive amount of validation going on here. `data_attributes` as an example doesn't validate the input. Any reason why not to validate the values here to check their types etc.? Or is there some magic validation code elsewhere that I am not aware of?

Reply:
I think that's a bug then. Passing garbage to `data_attributes` will probably make some parts of the app crash. It should have been a postgres string array as well. Too late now...